sim-administration/hss-adapter/src/integration-test/resources/hss-adapter-config.yaml

```diff
@@ -4,11 +4,13 @@ logging:
 
 hlrs:
   - name: Foo
+    type: WG2
```

Comment:
Maybe "vendor" would be a better name? But this of course depends on context...

Reply:
fixed.